pallets/parachain-staking/src/weights.rs

```diff
@@ -309,24 +292,22 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 	// Storage: ParachainStaking Total (r:1 w:1)
 	#[rustfmt::skip]
 	fn delegate(x: u32, y: u32, ) -> Weight {
-		(90_567_000 as Weight)
-			// Standard Error: 3_000
-			.saturating_add((310_000 as Weight).saturating_mul(x as Weight))
-			// Standard Error: 1_000
-			.saturating_add((204_000 as Weight).saturating_mul(y as Weight))
+		(114_601_000 as Weight)
+			// Standard Error: 11_000
+			.saturating_add((23_000 as Weight).saturating_mul(x as Weight))
+			// Standard Error: 4_000
+			.saturating_add((94_000 as Weight).saturating_mul(y as Weight))
```

Comment:
Does this change make sense? It's much cheaper (for large `x` and `y` values)

Reply:
`DelegatorState` shows up which got a size reduction (collator count bounded requests vector was moved out) so some reduction is expected. But from the benchmarking code of `delegate`, I do not see any delegation requests that would've been part of this bench (the vector will always be 0) - unless I missed something. So whatever we are looking at is either, noise or encode/decode gains.